commit 60092db3c1a7ff6caf9b7b7dd6fcdaa42d63a1b3
Author: Ingmar Vanhassel <ingmar@exherbo.org>
Date:   Wed May 27 06:13:52 2009 +0200

    Compatibility with CMake 2.6.4
    
    Since version 2.6.4, CMake enforces unique binary directories, so we
    can't use TRAVERSO_BUILD_DIR as global build directory anymore.

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index eaba0e3..2f83ade 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,11 +1,11 @@
-ADD_SUBDIRECTORY(engine ${TRAVERSO_BUILD_DIR})
-ADD_SUBDIRECTORY(audiofileio ${TRAVERSO_BUILD_DIR})
-ADD_SUBDIRECTORY(commands ${TRAVERSO_BUILD_DIR})
-ADD_SUBDIRECTORY(core ${TRAVERSO_BUILD_DIR})
-ADD_SUBDIRECTORY(3rdparty ${TRAVERSO_BUILD_DIR})
-ADD_SUBDIRECTORY(plugins ${TRAVERSO_BUILD_DIR})
-ADD_SUBDIRECTORY(sheetcanvas ${TRAVERSO_BUILD_DIR})
-ADD_SUBDIRECTORY(traverso ${TRAVERSO_BUILD_DIR})
+ADD_SUBDIRECTORY(engine)
+ADD_SUBDIRECTORY(audiofileio)
+ADD_SUBDIRECTORY(commands)
+ADD_SUBDIRECTORY(core)
+ADD_SUBDIRECTORY(3rdparty)
+ADD_SUBDIRECTORY(plugins)
+ADD_SUBDIRECTORY(sheetcanvas)
+ADD_SUBDIRECTORY(traverso)
 
 IF(USE_PCH)
     ADD_PRECOMPILED_HEADER(precompiled_headers precompile.h)
diff --git a/src/commands/CMakeLists.txt b/src/commands/CMakeLists.txt
index 614fefc..7c08fce 100644
--- a/src/commands/CMakeLists.txt
+++ b/src/commands/CMakeLists.txt
@@ -1,4 +1,4 @@
-ADD_SUBDIRECTORY(plugins ${TRAVERSO_BUILD_DIR})
+ADD_SUBDIRECTORY(plugins)
 
 INCLUDE_DIRECTORIES(
 ${CMAKE_SOURCE_DIR}/src/commands
diff --git a/src/commands/plugins/CMakeLists.txt b/src/commands/plugins/CMakeLists.txt
index f77d48d..188972b 100644
--- a/src/commands/plugins/CMakeLists.txt
+++ b/src/commands/plugins/CMakeLists.txt
@@ -1 +1 @@
-ADD_SUBDIRECTORY(TraversoCommands ${TRAVERSO_BUILD_DIR})
+ADD_SUBDIRECTORY(TraversoCommands)
